package.json

```diff
@@ -130,6 +136,11 @@
               }
             }
           },
+          "blender.defaultExecutableIndex": {
```

Comment:
I think that can just be a `default` boolean stored for the blender executable. I *think* that is more conventional. Haven't seen an index like that in vscode settings yet.

Reply:
I've changed this to use "isDefault" stored in each Blender executable, the command also prompts the user to select default if none is found.